snforge_std/src/cheatcodes/storage.cairo

```diff
@@ -2,12 +2,14 @@ use core::array::ArrayTrait;
 use core::traits::Into;
 use core::option::OptionTrait;
 use core::traits::TryInto;
-use starknet::{testing::cheatcode, ContractAddress, storage_address_try_from_felt252};
+use starknet::{testing::cheatcode, ContractAddress, StorageAddress};
 use core::panic_with_felt252;
 use super::super::_cheatcode::handle_cheatcode;
 
 fn validate_storage_address_felt(storage_address_felt: felt252) {
-    match storage_address_try_from_felt252(storage_address_felt) {
+    let result: Option<StorageAddress> = storage_address_felt.try_into();
```

Comment:
Note for the future: this is caused by this signature `pub(crate) extern fn storage_address_try_from_felt252`